apps/cinema-info-server/CinemaServer/Data/DTO/InterfaceDTO/IHallDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace CinemaServer.Data.DTO.InterfaceDTO
+{
+    public interface IHallDTO
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Address { get; set; }
+        public int Rows { get; set; }
+        public int Columns { get; set; }
+        public int AllPlaces { get; set; }
```

Comment:
??

Reply:
![image](https://user-images.githubusercontent.com/90874301/197345439-96682559-dca8-4dbc-bafe-b14a05cc3e3a.png)
Correct Table hall and interfaceDTO

Reply:
All places don't need